cmd/main.go

```diff
@@ -39,6 +39,10 @@ func main() {
 
 	go monitorSignals(sigs, quit, logger)
 
+	// Redis
+	redis := network.NewRedis(config.Redis.URL, logrus.Get("Redis"))
+	redisStartedChan := make(chan bool, 1)
```

Comment:
Just do this if you have another thing to change.
- Change the order just to keep the pattern of the code.

You can close this thread if necessary.

Reply:
Done.